src/pages/Lend/LendByCategoryPage.vue

```diff
@@ -708,8 +731,15 @@ export default {
 					client.query({ query: experimentQuery, variables: { id: 'mfi_recommendations' } }),
 					// experiment: VUE- Category Service driven FLSS channels
 					client.query({ query: experimentQuery, variables: { id: 'flss_category_service' } }),
+					// experiment: CORE-854 Loan Finding Page
+					client.query({ query: experimentQuery, variables: { id: 'loan_finding_page' } }),
 				]);
 			})
+				.then(() => {
+					return client.query({
+						query: mlOrderedLoanChannels
```

Comment:
Can this duplicate `mlOrderedLoanChannels` be removed?

Reply:
+1 dupe can be removed